docs/schema.yaml

```diff
@@ -0,0 +1,31 @@
+config_version: map(int(), str())
+
+general:
+  cm_device_codename: str()
+  cm_release_type: str()
+  cm_release_version: str()
+  template: str(required=False)
+
+repositories: list(include('repository'))
+
+apps: list(include('app'))
```

Comment:
From what I can tell the `misc/MyAppList/misc-apps.xml` file needs the a human readable name.

For apps from the F-Droid repositories we will be able to get the application name from the index.xml file, but not for remote files. I would like to also include a `name: str()` here:
```
app_remote:
  id: str()
  install_type: enum('system', 'privileged', 'user', required=False)
  name: str()
  url: str()
```

This is required to fix: #61